js/src/layers/Velocity.js

```diff
@@ -25,7 +25,7 @@ export class LeafletVelocityModel extends layer.LeafletLayerModel {
       minVelocity: 0,
       maxVelocity: 10,
       velocityScale: 0.005,
-      colorScale: []
+      _colorScale: []
```

Comment:
Actually the `VelocityLayer` expects [colorScale](https://github.com/onaci/leaflet-velocity/blob/ab97f77c237e60ff7bb1f5bb30335c6e583d9209/src/js/L.VelocityLayer.js#L10), so this won't work, unless you copy the value of `_colorScale` to the `colorScale` option.

Reply:
We have reverted the change of making color_scale a private attribute.